tests/Report/junit.xsd

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
next thing - this xsd file is subset of one from jenkins or it is exactly the same file ?